R/db-helpers.R

```diff
@@ -88,31 +84,7 @@ queries_set_pk_cols <- function(dest, pk_information) {
 }
 
 queries_adapt_fk_col_classes <- function(dest, fk_information) {
-  db_child_tables <- fk_information$db_child_table
-  cols_to_adapt <- fk_information$child_fk_col
-  child_col_classes <- fk_information$col_class
-  cols_db_classes <- class_to_db_class(dest, child_col_classes)
-  if (is_mssql(dest)) {
-    pmap_chr(
-      list(
-        db_child_tables,
-        cols_to_adapt,
-        cols_db_classes
-      ),
-      ~ glue("ALTER TABLE {..1} ALTER COLUMN {..2} {..3}")
-    )
-  } else if (is_postgres(dest)) {
-    pmap_chr(
-      list(
-        db_child_tables,
-        cols_to_adapt,
-        cols_db_classes
-      ),
-      ~ glue("ALTER TABLE {..1} ALTER COLUMN {..2} TYPE {..3}")
-    )
-  } else {
-    return("")
-  }
+  ""
```

Comment:
Your approach here once more works for Postgres, but I am quite certain that for MSSQL it will fail (this was most likely the reason why I added these... but we can leave it out for Postgres)